test/core/mldataset/test_base.py

```diff
@@ -5,6 +5,8 @@
 from xcube.core.tilingscheme import TilingScheme
 from .helpers import get_test_dataset
 
+_CRS84 = 'OGC:CRS84'
```

Comment:
@pont-us do we have a convention for naming constants? I feel like it is the first time I see a "_" prior to a constant name, but I might be wrong.

Reply:
The leading underscore denotes constants that are only used within the module in which they are declared. This is used in some places in xcube and the plugins. I did not see any rule in our conventions that en- or discourages its use. Personally, I like it as it gives some order.

Reply:
The leading underscore is recommended in PEP-8, so implicitly part of our conventions :)